app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_like_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
**[OPTIONAL]**

- [ ] It will be nice to use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to the database, then make the update_post_counter method private